.github/workflows/simulation.yml

```diff
@@ -203,6 +203,37 @@ jobs:
     - name: Codecov
       uses: codecov/codecov-action@v1
 
+  external-commissioner:
+    continue-on-error: true
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Bootstrap
+      run: |
+        sudo apt-get --no-install-recommends install -y ninja-build
+        git clone https://github.com/openthread/ot-commissioner.git /tmp/ot-commissioner
+    - name: Build
+      run: |
+        cd /tmp/ot-commissioner
+        script/bootstrap.sh
+        cmake -GNinja                           \
+              -DCMAKE_CXX_STANDARD=11           \
+              -DCMAKE_CXX_STANDARD_REQUIRED=ON  \
+              -DCMAKE_BUILD_TYPE=Release        \
+              -DCMAKE_INSTALL_PREFIX=/usr/local \
+              -DOT_COMM_COVERAGE=ON             \
+              -S . -B build
+        cmake --build build
+        sudo cmake --install build
+    - name: Run
+      run: |
+        export OPENTHREAD="$(pwd)"
```

Comment:
Didn't the previous job executed `cd /tmp/ot-commissioner`? Is $(pwd) still the openthread directory?

Reply:
`PWD` is changed back before starting each stage.